src/content/en/fundamentals/performance/optimizing-javascript/tree-shaking/index.md

```diff
@@ -0,0 +1,423 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Knowing where to begin optimizing your application's JavaScript can be daunting. If you're taking advantage of modern tooling such as webpack, however, tree shaking might be a good place to start!
+
+{# wf_updated_on: 2018-06-12 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_blink_components: Blink>JavaScript #}
+
+# Reduce JavaScript Payloads with Tree Shaking {: .page-title }
+
+{% include "web/_shared/contributors/jeremywagner.html" %}
+
+Today's web applications can get pretty big, especially the JavaScript part of
+them. [As of
+mid-2018](https://httparchive.org/reports/state-of-javascript#bytesJs), HTTP
```

Comment:
Nit: Put the link on the specific content it refers to. Right now you're linking to "mid-2018", but the content is really about the "median transfer size of JavaScript on mobile"